src/components/map/layers/PopupLayer.tsx

```diff
@@ -31,12 +31,10 @@ class PopupLayer extends Component<PopupLayerProps> {
             const node = this.props.popupStore!.popupNode as INode;
 
             const openNode = () => {
-                // TODO deal fetching nodeId this in the endpoint
-                this.props.sidebarStore!.setOpenNodeId(node.id);
```

Comment:
I don't quite understand the Todo, what does "nodeId this in the endpoint" mean?

Reply:
I guess it was written by VV and it means to read nodeId from url params instead of adding it to store when clicking "Avaa solmu" from popup